rpc/ethereum/backend/backend.go

```diff
@@ -704,10 +705,10 @@ func (e *EVMBackend) SendTransaction(args types.SendTxArgs) (common.Hash, error)
 	// NOTE: If error is encountered on the node, the broadcast will not return an error
 	syncCtx := e.clientCtx.WithBroadcastMode(flags.BroadcastSync)
 	rsp, err := syncCtx.BroadcastTx(txBytes)
-	if err != nil || rsp.Code != 0 {
-		if err == nil {
-			err = errors.New(rsp.RawLog)
-		}
+	if rsp != nil && rsp.Code != 0 {
```

Comment:
is this correct?

In original behavior, if err != nil, then we return err

Reply:
I think normally `err` and `rsp` won't be nil or non-nil at the same time.